### PR TITLE
Ensure log output is in JSON format

### PR DESCRIPTION
**Pull request by ijdickinson**
epimorphics/hmlr-linked-data#87

Ensure that all log output from the landing page app is written in JSON
format.

- use `json-rails-logger` as the log formatter
- ensure Rails and Puma startup messages in JSON

### CODE REVIEW

**Review by der — APPROVED** [review]
Impressive dedication to even patch the rails startup!
My knowledge of rails puma/rails config is not enough to make any useful comments on this one but seems fine to me.

File: config/application.rb [hunk 1]
```diff
@@ -18,6 +18,28 @@
 
 require 'qonsole_rails'
 
+# Monkey-patch the bit of Rails that emits the start-up log message, so that it
+# is written out in JSON format that our combined logging service can handle
+# This versio is Rails 5.x specific. A different pattern is needed for Rails 6
```
Comment (jonrandahl) [inline config/application.rb]: @ijdickinson small typo 😃

**Review by jonrandahl — APPROVED** [review]
Other than the small typo LGTM! 👍

**Review by andrew-pickin-epi — COMMENTED** [review]
The required time format is more specific that just rfc3339.
What format does `DateTime.now.rfc3339(3)` produce?

File: config/application.rb (hunk 1 above)
Comment (ijdickinson) [reply config/application.rb]: Thanks, fixed

**Comment by ijdickinson** [comment]
> What format does `DateTime.now.rfc3339(3)` produce?

@andrew-pickin-epi:

```
irb(main):003:0> DateTime.now.rfc3339(3)
=> "2022-04-05T15:59:19.445+01:00"
```

**Comment by ijdickinson** [comment]
I need to merge this now, as it's holding up other progress. We can add a specific ticket if we need to to update the timestamp format

**Pull request merged**